README.md

```diff
@@ -0,0 +1,154 @@
+
+The present dynamic model implements the formulation proposed mainly by [Fossen][].
+
+It implements the dynamic and kinematic of a vehicle and its hydrodynamic effect. See [UWV Parameters](#parameters) section.
+There are three induced hydrodynamic efforts:
+
+ 1. Added mass due to the inertia of the surrouding fluid.
+ 2. Damping effect.
+ 3. Restoring gravitational forces (weight and buoyancy).
+
+The hydrodynamic effect is embedded in the parameters passed to the model.
+It is required that the parameters are identified previously. (The code doesn't estimated them).
+
+In order to add some flexibility on how each term would affect the model, the present library has 
+a ModelType property. It allows the user to choose a specific formulation of the damping term or 
+to ignore the Corilois effect for a simplified model, for example. See [UWV Parameters](#parameters) section.
+
+The control efforts applied in the vechile are considered to be in the CoG.
+In case a set of propellers need to be represented, the dynamic model just requires
+the resultant effect (in terms of forces and torques) in the CoG.
+
+In the 6 degree of freedom (DOF) formulation, the order of representing each DOF is:
+
+ 1. linear x
+ 2. linear y
+ 3. linear z
+ 4. angular x
+ 5. angular y
+ 6. angular z
+
+The efforts applied as well as the velocities (both linear as angular) are represented in body-frame.
+The position and the orientation are represented is fixed world-frame.
+ 
+As the main focus of the present library is the dynamic effect, it is possile to make the simulator ignores 
+the kinematic part of the formulation. ModelSimulator defines if the integration from velocities to pose and
+orientation would be done or not. See [Simulation](#simulation) section.
+
+## UWV Parameters <a id="parameters"></a>
+
+The parameters that need to be set for the dynamic model are:
+
+- Model Type
+- Inertia Matrix
+- Damping Matrices
+- Distance Body to Center of Buoyancy
+- Weight
+- Buoyancy
+
+### Model Type
+ 
+Model Type defines which effects will be present in the model.
+At the moment, three Model Types are defined:
+
+ 1. SIMPLE: 
+	 It ignores the Coriolis effect and consider the Damping Matrices with two terms,
+	the fist the linear damping and the second quadratic damping.
+	It is based on [Smallwood][].
+
+	> **Note:**
+	For considering a decoupled model, one must set the matrices of inertia and damping as diagonal.
+
+
+ 2. COMPLEX:
+	It consider the Coriollis effect and a Damping Matrices with six terms,
+	all quadratic damping, been relative to the 3 linear and 3 angular DOF respectivelly.
+	It is based on [McFarland][].
+	
+ 3. INTERMEDIATE:
+	It consider the Coriollis effect and a Damping Matrices with two terms,
+	the first the linear and the second the quadratic damping.
+
+Other ways to formulate the model could be defined.
+
+### Inertia Matrix
+
+The $6 \times 6$ inertia matrix consider both the inertia of the vehicle and the added mass.
```

Comment:
>The $6 \times 6$ inertia matrix

LaTeX formatting style seems not to be working here, thus it's making the text confuse